pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -911,7 +934,7 @@ func TestBuildConfiguration(t *testing.T) {
 	assert.Equal(t, "https://maven.mirror.com/", server.BuildConfigs[0].Spec.Strategy.SourceStrategy.Env[1].Value)
 	assert.Equal(t, "dir", server.BuildConfigs[0].Spec.Strategy.SourceStrategy.Env[2].Value)
 	assert.Equal(t, "s3cr3t", server.BuildConfigs[0].Spec.Triggers[0].GitHubWebHook.Secret)
-	assert.NotEmpty(t, server.BuildConfigs[0].Spec.Triggers[1].GenericWebHook.Secret)
+	assert.Empty(t, server.BuildConfigs[0].Spec.Triggers[1].GenericWebHook.Secret)
```

Comment:
why should this be `NotEmpty` by default?

Reply:
@ruromero @bmozaffa thoughts?

Reply:
IIRC it is generated by default to allow external apps to use this secret when sending webhook events. The buildConfig is the owner of the secret so it makes sense to have always a generated one by default

Reply:
But this should apply to the defined Webhook. In this case as the GitHubWebhook is defined I agree the secret for the GenericWebhook should not be generated.